dev-project/pipelinewise-config/tap_postgres_logical.yml

```diff
@@ -22,9 +22,6 @@ db_conn:
   user: "pipelinewise"                 # PostfreSQL user
   password: "secret"                   # Plain string or vault encrypted
   dbname: "postgres_source_db"         # PostgreSQL database name
-  logical_poll_total_seconds: 5        # Seconds to wait before assuming no more data
```

Comment:
Why do you need to take this out ?

If you take this out, the default is ~30 minutes
Meaning the automated tests could take 30 minutes to fail in some cases

Reply:
I'm taking it out because it's a duplicate, the key is already set to 3 [here](https://github.com/transferwise/pipelinewise/blob/9fd886038cc19ed62ea200ba1f4f5d72275624c4/dev-project/pipelinewise-config/tap_postgres_logical.yml#L17).